game/src/main/scala/io/aigar/game/Player.scala

```diff
@@ -33,4 +35,11 @@ class Player(val id: Int, startPosition: Vector2) {
   def isActive():Boolean = {
     cells.exists(_.behavior.isActive)
   }
+
+  /**
+    * Removes dead cell from the player's cell list
+    */
+  def removeCell(cell: Cell): Unit ={
+    cells = cells diff List(cell)
```

Comment:
Why not using the same `filterNot` as you already did? Or simply use the same thing both places